util/app-config/src/configs/network.rs

```diff
@@ -105,7 +116,14 @@ impl Config {
             .create(true)
             .write(true)
             .open(path)
-            .and_then(|mut file| file.write_all(&random_key_pair))
+            .and_then(|mut file| {
+                #[cfg(unix)]
```

Comment:
Does it also cover mac?

Reply:
`#[cfg(unix)]` contains mac

`cfg(target_os = "linux")` does not contain mac